firebase-installations/src/main/java/com/google/firebase/installations/remote/FirebaseInstallationServiceClient.java

```diff
@@ -158,15 +162,17 @@ public InstallationResponse createFirebaseInstallation(
         writeFIDCreateRequestBodyToOutputStream(httpURLConnection, fid, appId);
 
         int httpResponseCode = httpURLConnection.getResponseCode();
+        requestLimiter.setNextRequestTime(httpResponseCode);
 
         if (httpResponseCode == 200) {
           return readCreateResponse(httpURLConnection);
         }
 
         logFisCommunicationError(httpURLConnection, appId, apiKey, projectID);
 
-        if (httpResponseCode == 429 || (httpResponseCode >= 500 && httpResponseCode < 600)) {
+        if (httpResponseCode >= 500 && httpResponseCode < 600) {
```

Comment:
What happened to `429`?
We `logBadConfigError` for that status now? 
If that is the case, I think it's not correct.

Reply:
Oh right. Changed it to throw an exception on 429.